docs/api/group-feed-shopping-api.md

```
@@ -0,0 +1,744 @@
+| 값 | 설명 |
+|----|------|
+| `USER_CREATED` | 사용자 생성 피드 |
+| `DAILY_MENU_NOTIFICATION` | 오늘의 식단 알림 |
+| `NEW_RECIPE_ADDED` | 새 레시피 추가 알림 |
+
+---
+
+## Group API
+
+### 1. 그룹 생성
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)